### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by jo-asakura**
:eyeglasses:  @ajacksified or @danehansen

### CODE REVIEW

File: assets/less/components/listing.less [hunk 1]
```diff
@@ -110,26 +110,31 @@
 }
 
 .well.listing-selftext {
-   background-color: #fafafa;
-   border: none;
-   padding: 1em;
+  background-color: #fafafa;
```
Comment (ajacksified) [inline assets/less/components/listing.less]: Let's pull this out into a variable.

File: src/lib/danehansen/Utils.es6.js [hunk 2]
```diff
@@ -473,11 +473,11 @@ var Utils={};
 
 	Utils.touch=function()
 	{
-		if(_touch==null)
-			_touch="ontouchstart" in window;
-		return _touch;
+    if (typeof window !== 'undefined') {
+      return ("ontouchstart" in window);
```
Comment (ajacksified) [inline src/lib/danehansen/Utils.es6.js]: single quotes

File: src/views/components/Listing.jsx [hunk 3]
```diff
@@ -271,8 +271,8 @@ class Listing extends React.Component {
     }
 
     var extImageSource;
-    if (this.state.expanded || this.props.single) {
-      var url = isImgurDomain(listing.domain) ? listing.url.replace(imgMatch, '') : listing.url;
+    if ((this.state.expanded || this.props.single) && isImgurDomain(listing.domain)) {
```
Comment (ajacksified) [inline src/views/components/Listing.jsx]: why imgur only?

File: assets/less/components/listing.less (hunk 1 above)
Comment (jo-asakura) [inline assets/less/components/listing.less]: It's not me.. it was there, I just shifted it to be 2 spaces instead of 3 :smile:

**Comment by ajacksified** [comment]
:nail_care:

File: assets/less/components/listing.less (hunk 1 above)
Comment (ajacksified) [inline assets/less/components/listing.less]: Fair enough.

File: src/views/components/Listing.jsx (hunk 3 above)
Comment (jo-asakura) [inline src/views/components/Listing.jsx]: because Daniel told so :smile:

**Comment by jo-asakura** [comment]
:haircut: @ajacksified

**Comment by ajacksified** [comment]
:fish:

**Pull request closed without merging**